.gitignore

```diff
@@ -1,2 +1,3 @@
 .npm
 .coverage
+node_modules
```

Comment:
Yes it is, I added a package.json with a few scripts to run, I've also installed eslint and precommit-hook as devDependencies